driver-reactive-streams/src/test/functional/com/mongodb/reactivestreams/client/RewrapManyDataKeyProseTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.reactivestreams.client;
+
+import com.mongodb.ClientEncryptionSettings;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.client.AbstractRewrapManyDataKeyProseTest;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.vault.ClientEncryption;
+import com.mongodb.client.vault.ClientEncryptions;
+import com.mongodb.reactivestreams.client.syncadapter.SyncMongoClient;
+
+public class RewrapManyDataKeyProseTest extends AbstractRewrapManyDataKeyProseTest {
```

Comment:
Rename to `ClientEncryptionRewrapManyDataKeyProseTest` to aid future discovery

Reply:
Many tests are named `AbstractClientSideEncryption...` - is this just a minor inconsistency, or is there some difference?